bin/reth/src/stage/mod.rs

```diff
@@ -136,11 +143,14 @@ impl Command {
                     });
                 }
 
-                let p2p_secret_key = get_secret_key(&self.p2p_secret_key)?;
+                let default_secret_key_path = data_dir.p2p_path().p2p_secret_path();
+                let p2p_secret_key = get_secret_key(&default_secret_key_path)?;
```

Comment:
cut the `reth/p2p/secret` dir format and just call it `reth/discovery-secret` and `reth/known-peers.json`, no need to have both `p2p_path` and `net_path`, it's unnecessary

Reply:
done